recipe/recipe.yaml

```diff
@@ -0,0 +1,310 @@
+schema_version: 1
+
+context:
+  version: 1.0.0
+
+recipe:
+  name: gymnasium-split
+  version: ${{ version }}
+
+source:
+  url: https://github.com/Farama-Foundation/Gymnasium/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: d61a54c70d1960a0b64c63fe87edc30a0bac09551c27d8d94c36c54bf5fae5d9
+  patches:
+    # backport https://github.com/Farama-Foundation/Gymnasium/pull/1222
+    - patches/1222.patch
+    # backport https://github.com/Farama-Foundation/Gymnasium/pull/1361
+    - patches/1361.patch
+
+build:
+  number: 2
+
+outputs:
+  - package:
+      name: gymnasium
+    build:
+      script: ${{ PYTHON }} -m pip install . -vv --no-deps
+    requirements:
+      build:
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - numpy
+      host:
+        - python
+        - pip
+        - setuptools
+        - cloudpickle >=1.2.0
+        - numpy
+      run:
+        - python
+        - cloudpickle >=1.2.0
+        - farama-notifications
+        - jax-jumpy >=1.0.0
+        - typing_extensions >=4.3.0
+    tests:
+      - python:
+          imports:
+            - gymnasium
+
+  - package:
+      name: gymnasium-all
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - ${{ pin_subpackage('gymnasium-atari', exact=True) }}
+        - ${{ pin_subpackage('gymnasium-box2d', exact=True) }}
+        - ${{ pin_subpackage('gymnasium-classic_control', exact=True) }}
+        - ${{ pin_subpackage('gymnasium-mujoco', exact=True) }}
+        # see below
+        # - {{ pin_subpackage('gymnasium-mujoco_py', exact=True) }}
+        - ${{ pin_subpackage('gymnasium-toy_text', exact=True) }}
+        - ${{ pin_subpackage('gymnasium-other', exact=True) }}
+    tests:
+      - python:
+          pip_check: true
+          imports:
+            - gymnasium
+      - files:
+          source:
+            - tests/
+        requirements:
+          run:
+            - mock
+            - pytest
+            - scipy
+            - dill
+            - pip
+            - if: (target_platform == "osx-64" or target_platform == "linux-64") and match(python, "<3.13")
+              then:
+                - jax >=0.4,<0.5.0
```

Comment:
not enthusiastic about this, but hoping we can get rid of the cap once we get back up to current gymnasium versions.